tests/e2e/setup/400-morose-init-files.ts

```diff
@@ -0,0 +1,42 @@
+import * as fs from '../utils/fs';
+import { generateHash } from '../../../src/api/auth';
+
+export default function() {
+  let authPath = 'morose/auth.json';
+  let configPath = 'morose/config.json';
+
+  let secret = Math.random().toString(36).substr(2, 10);
+  let pass = generateHash('blabla', secret);
+  let config = {
+    port: 10000,
+    ssl: false,
+    secret: secret,
+    upstream: 'https://registry.npmjs.org',
+    useUpstream: true,
+    saveUpstreamPackages: true
+  };
+  let auth = {
+      organizations: [
+        {
+          name: 'bleenco',
+          teams: [ { name: 'developers', members: [ 'admin' ] } ],
+          members: [ { name: 'admin', role: 'owner' } ],
+          packages: []
+        }
+      ],
+      users: [
+        { name: 'admin', password: pass, fullName: '', email: '', tokens: [] },
+        { name: 'developer', password: pass, fullName: '', email: '', tokens: [] }
+      ],
+      packages: []
+    };
+
+  return Promise.resolve()
+    .then(() => fs.createDir('morose'))
+    .then(() => {
```

Comment:
```ts
.then(() => fs.writeJsonFile(configPath, config)
.then(() => fs.writeJsonFile(authPath, auth)
```
this is nicer, I think. 
When using arrow function `=>` you can use oneliners, they return by default, so:
```ts
.then(() => {
  return fs.writeJsonFile(configPath, config);
})
```
is exactly same as
```ts
.then(() => fs.writeJsonFile(configPath)
```